tests/rptest/services/redpanda.py

```diff
@@ -585,6 +585,12 @@ def __init__(self,
         # stash a copy here so that we can quickly look up e.g. addresses later.
         self._node_configs = {}
 
+        self._seed_servers = [self.nodes[0]]
```

Comment:
is this intended ? it seems that it cause some tests to fail

Reply:
yes. The template for that is:
```
{% if include_seed_servers %}
  seed_servers:
  {% for node in seed_servers %}
    - host:
        address: {{node.account.hostname}}
        port: 33145
  {% endfor %}
{% endif %}
```
and `include_seed_servers` is initialized as
```
        include_seed_servers = True
        node_id = self.idx(node)
        if omit_seeds_on_idx_one and node_id == 1:
            include_seed_servers = False
```
so the quoted line will make `seed_servers` a root node everywhere except for the root node (for root-driven bootstrap)

Reply:
actually as @andrwng has pointed out, this will make `PartitionBalancerTest.test_full_nodes` fail because that test creates a faux `Redpanda` instanse to skip actual testing in debug. I'm posting a fix.